docs/DesignDocs/libc_pthread_proposal.md

```diff
@@ -0,0 +1,80 @@
+# To support pthread in Open Enclave SDK
+
+Currently, pthread management APIs are not implemented in OE SDK while they are
+implemented in SGX SDK. The pthread APIs are added when porting MKL-DNN to SGX SDK.
+To get better performance, MKL-DNN should be configured to build with OpenMP which
+depends on pthread.
+
+## New APIs
```

Comment:
```suggestion
## New Internal APIs
```
Important point is that these are not public APIs (the pthread* APIs are the public API) and can change at any time.